src/styles/container.scss

```diff
@@ -0,0 +1,21 @@
+$center-size: 15px;
+$watch-size: 500px;
+$width-hand: 3px;
+
+@mixin round-block ($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin height-hand ($height, $color) {
```

Comment:
I would change this mixin name, because you use not only height inside. It can be confused
Probably, i don't even use mixin for such changes